### DISCUSSION TITLE
How to hide the uri of adapter.vcfGzLocation in the jbrowse2 config?

### DISCUSSION THREAD

**Discussion by Bobo22331** [GMOD/jbrowse-components#3915] [Q&A] [answered]
Hi,
How to hide the uri of adapter.vcfGzLocation in the jbrowse2 config?
Such as the following part:
<img width="788" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/19387444/9bcae9b5-b5ec-462e-a6b0-0accd2b2e3f3">

<img width="1313" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/19387444/14d69be2-7a1c-4979-9a68-872f6f555f51">

**Comment by cmdcolin** [comment, marked as answer]
you can add formatAbout:{hideUris:true} to your track config

an example is here

https://github.com/GMOD/jbrowse-components/blob/4b2938d5e9b652b78916d95861a6946b07ad06e7/test_data/volvox/config.json#L620-L642

**Reply by cmdcolin** [reply]
note that the URI would still be visible to someone who was snooping in their browser's network logs so this param just hides it from the GUI. hope that helps
